### DISCUSSION TITLE
Simplest way to execute a dynamic command on join?

### DISCUSSION THREAD

**Discussion by FinnPaes** [ORelio/Minecraft-Console-Client#1860] [Q&A]
So I am working on a Discord bot that is able to join a Minecraft server for you. (Sends a post request to a Linux server that joins the server for you).

Now, I want to make it possible to also send a command with it that executes when the bot joins the server, without leaving. It's not really a great way to use the tasks.ini file, cause that's not dynamic. Is there any way to make it dynamic? Like a parameter on the _MinecraftClient.exe username password IP "command"_ and not leave the server?

Thanks

**Comment by ORelio** [comment]
You can use the Script (Task) scheduler:

https://github.com/MCCTeam/Minecraft-Console-Client/tree/master/MinecraftClient/config#using-the-script-scheduler--task-scheduler

> Using the Script Scheduler / Task Scheduler
> ------
> 
> The script scheduler allows you to perform scripts or internal commands on various events such as log on server, time interval, or fixed > date and time. Simply enable the ScriptScheduler bot and specify a tasks file in your INI file. Please read [`sample-tasks.ini`](https://github.com/MCCTeam/Minecraft-Console-Client/blob/master/MinecraftClient/config/sample-tasks.ini) for learning how to make your own task file.

However it works using a configuration file instead of command-line arguments.
